obspy/core/tests/test_waveform_plugins.py

```diff
@@ -151,12 +151,12 @@ def test_read_and_write(self):
                         self.assertEqual(st[0].stats.starttime, start)
                         if format == 'AH':
                             self.assertEqual(st[0].stats.delta, 0.25)
-                            self.assertEqual(st[0].stats.endtime,
-                                                start + 499.75)
+                            cend = start + 499.75
+                            self.assertEqual(st[0].stats.endtime, cend)
```

Comment:
just FYI, would've also worked to just align it under the bracket.. that's what PEP8 was complainign about
```
                            self.assertEqual(st[0].stats.endtime,
                                             start + 499.75)
```